src/components/Table/index.js

```diff
@@ -160,6 +177,8 @@ const Table = () => {
                   >
                     Reset
                   </Button>
+                  <br />
+                  <br />
```

Comment:
These `<br />` elements look like they're used to add whitespace between the existing `Reset` button and the new `Checked`/`Unchecked` dropdown list.

We can instead remove these two lines and update some CSS in [`src/components/Table/styles.scss`](https://github.com/seanprashad/leetcode-patterns/blob/fb8b26e9e0f0319b31fb9c1838a32a7bdfacd995/src/components/Table/styles.scss#L48-L51) to make things more straight-forward:

```css
.reset-button {
  margin-top: 10px;
  margin-bottom: 10px;
  font-size: 0.7rem;
}
```